langchain/chains/base.py

```diff
@@ -268,7 +268,7 @@ def dict(self, **kwargs: Any) -> Dict:
         """Return dictionary representation of chain."""
         if self.memory is not None:
             raise ValueError("Saving of memory is not yet supported.")
-        _dict = super().dict()
+        _dict = super().dict(exclude={"callbacks"})
```

Comment:
this should be unnecessary after https://github.com/hwchase17/langchain/pull/4364

Reply:
great thanks! BTW for simplicity i kept the agent saving logic caught by the NotImplementedError but Agents + outputparser seem to have a similar serialization issue.